src/core/containers/ServerHtml.js

```diff
@@ -99,6 +99,7 @@ export default class ServerHtml extends Component {
         <head>
           <meta charSet="utf-8" />
           <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <meta name="twitter:card" content="summary" />
```

Comment:
What is this doing? This is just using the text "summary" from what I can tell. We want the add-on information to be shown, so I think you'll need to refer to the docs in https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started more, and you may want to find a way to test this...